investpy/__init__.py

```diff
@@ -2480,9 +2682,11 @@ def search_etfs(by, value):
     search_result = etfs.loc[etfs['matches'] == True].copy()
```

Comment:
All the changes done not related with `cross_currencies` should not be included since the previous code was properly formatted and the published changes are just modifications on how the data is tabulated or presented to the programmer. So **just the additions related to cross currencies data retrieval will be taken into consideration**, unless changes in any other function improves it.